R/rcloud.shiny.R

```diff
@@ -1,12 +1,17 @@
 rcloud.proxy.url <- function(port, search, hash) {
   info <- rcloud.session.info()
-  paste0('proxy.R/', info$user, '/', info$id, ':', port, '/', search, hash)
+  paste0('/proxy.R/', info$user, '/', info$id, ':', port, '/', search, hash)
 }
 
 rcloud.shinyApp <- function(ui, server, options) {
-  library(rcloud.web)
-  library(shiny)
-
+  if(!requireNamespace("rcloud.web", quietly = TRUE)) {
+    stop("'rcloud.web' is needed for this function to work. Please install it.",
+         call. = FALSE)
+  }
+  if(!requireNamespace("shiny", quietly = TRUE)) {
+    stop("'shiny' is needed for this function to work. Please install it.",
+         call. = FALSE)
+  }
   appHandlers <- NULL
```

Comment:
I forgot that we were already loading the libraries lazily (inside the functions). So I don't think this is necessary; I just wanted to be sure we could load rcloud.shiny as an RCloud add-on for the view types menu, without loading shiny.

It also seems not to bring names into scope (maybe it would need `attachNamespace` as well) so I was seeing errors like `fluidPage not found` . So I'll just revert this one diff.